config/software/openssl.rb

```diff
@@ -23,7 +23,7 @@
 dependency "cacerts"
 dependency "openssl-fips" if fips_mode?
 
-default_version "1.0.2y"
+default_version "1.1.1k"
```

Comment:
Please leave default as `1.0.2y`

Reply:
Please add the `# do_not_auto_update` comment so we know not to update the default version.

Reply:
I was testing if the patch is working for other openssl versions like 1.1.1k but it is failing https://buildkite.com/chef/chef-omnibus-toolchain-master-omnibus-adhoc/builds/542 .

I have updated the default version as 1.0.2y again and also added # do_not_auto_update  comment.